build.sbt

```diff
@@ -235,19 +241,33 @@ ThisBuild / publishMavenStyle := true
 ThisBuild / bintrayOrganization := Some("projectglow")
 ThisBuild / bintrayRepository := "glow"
 
+val stableVersion = settingKey[String]("Stable version")
+ThisBuild / stableVersion := IO.read((ThisBuild / baseDirectory).value / "stable-version.txt").trim()
+
+lazy val stagedRelease = (project in file("core")).settings(
```

Comment:
Does this not mean that we're using the managed source directory from `core`? I don't think we want that, do we?

Reply:
Not quite - by setting `unmanagedSourceDirectories in Compile := Nil`, we are not using any code for compilation.

Reply:
To clarify, `unmanagedSourceDirectories` consists of `scalaSource` and `javaSource`: https://www.scala-sbt.org/1.x/docs/Howto-Customizing-Paths.html#Add+an+additional+source+directory

Reply:
Ah, got it. Why do we copy the settings from `core` then? It might be more clear to just base this on `commonSettings`